frontend/src/pages/SignUp.jsx

```diff
@@ -33,9 +33,7 @@ export default function SignUp() {
 			await register(email, password);
 			navigate("/dashboard");
 		} catch (error) {
-			alert(
-				error.message
-			);
+			alert(error.message);
```

Comment:
This error message currently is: `Password should be at least 8 characters. Password should contain at least one character of each: abcdefghijklmnopqrstuvwxyz, ABCDEFGHIJKLMNOPQRSTUVWXYZ, 0123456789, !@#$%^&*()_+-=[]{};\'\:"|<>?,./`~.`
Instead, could you change it to something more readable? For example, 
```
Password should contain:
- At least 8 characters
- At least one capital letter
- At least one lowercase letter
- At least one number
- At least one of !@#$%^&*()_+-=[]{};\'\:"|<>?,./`~

Don't need to do this right now but once more of the frontend designs are released we should probably have the password check be displayed directly on the form and responding to user input changes, rather than only checking on submit.